### ISSUE TITLE
auth-module-getters 

### ISSUE THREAD

**Issue by andysay** [feathersjs-ecosystem/feathers-vuex#361]
i think need add to main code this

```js
/*
eslint
@typescript-eslint/explicit-function-return-type: 0,
@typescript-eslint/no-explicit-any: 0
*/
export default function makeAuthGetters() {
  return {
    // Auth specific "Getters" will go here
    isAuthenticated(state) {
      return !!state.user;
    },
  }
}

```

**Comment by J3m5** [comment]
I use this one all the time!

**Comment by marshallswain** [comment]
I'll accept a PR that conditionally adds the `isAuthenticated` getter based on the presence of the `userService` option:

```
export default function makeAuthGetters(options) {
  const getters = {}

  // If a userService was provided, add an isAuthenticated getter.
  if (options.userService) {
    getters.isAuthenticated = function isAuthenticated(state) {
      return !!state.user
    }
  }

  return getters
}
```

A test would be really appreciated, but I won't reject the PR without it. ;)

**Comment by marshallswain** [comment]
This has been released in `feathers-vuex@3.2.0` along with a new `user` getter which is reactive.  The `user` state is now deprecated and will be removed in version 4.
